### ISSUE TITLE
Table element Hide label Issue

### ISSUE THREAD

**Issue by Bnayak123** [formio/formio.js#5175]
Environment:
Hosting type : Form.io

Version: Latest version (Even in online [demo builder](https://formio.github.io/angular-demo/#/forms/builder))

Browser: Chrome 
Version 113.0.5672.126 (Official Build) (64-bit)

Steps to Reproduce

 1. select the table component then add one label name and add some components to the table .

 2. select the modalEdit true and select the hideLabel true

 3. while rendering the component the label is not hidden.

Expected behavior

while rendering the component if we select hideLabel true label should be hidden.

JSON
{
  "display": "form",
  "components": [
    {
      "label": "Table Test",
      "cellAlignment": "left",
      "hideLabel": true,
      "modalEdit": true,
      "key": "tableTest",
      "type": "table",
      "input": false,
      "tableView": false,
      "rows": [
        [
          {
            "components": [
              {
                "label": "Text Field Test",
                "applyMaskOn": "change",
                "tableView": true,
                "key": "textFieldTest",
                "type": "textfield",
                "input": true
              }
            ]
          },
          {
            "components": []
          },
          {
            "components": []
          }
        ],
        [
          {
            "components": []
          },
          {
            "components": [
              {
                "label": "Number Test",
                "applyMaskOn": "change",
                "mask": false,
                "tableView": false,
                "delimiter": false,
                "requireDecimal": false,
                "inputFormat": "plain",
                "truncateMultipleSpaces": false,
                "key": "numberTest",
                "type": "number",
                "input": true
              }
            ]
          },
          {
            "components": []
          }
        ],
        [
          {
            "components": []
          },
          {
            "components": []
          },
          {
            "components": [
              {
                "label": "Time Test",
                "tableView": true,
                "key": "timeTest",
                "type": "time",
                "input": true,
                "inputMask": "99:99"
              }
            ]
          }
        ]
      ]
    },
    {
      "type": "button",
      "label": "Submit",
      "key": "submit",
      "disableOnInvalid": true,
      "input": true,
      "tableView": false
    }
  ]
}
![Table element Hide label Issue](https://github.com/formio/formio.js/assets/113301614/7516fb0f-c58f-484a-b717-4901d248868e)

**Comment by travist** [comment]
I agree this is a bug. It is probably in the template which is here https://github.com/formio/formio.js/blob/master/src/templates/bootstrap5/modalPreview/form.ejs.  We will log this as a nit bug, but if you wish to expedite the resolution of this, then please feel free to submit a pull request with a label hidden flag check in this template so that it does not show the label if the hide label flag is set.

**Comment by Sidiro23** [comment]
Thank you for reporting this issue. I can reproduce it and have logged a ticket for the backlog. We are always willing to review any contributions to resolve this issue from the open-source community.

For internal reference: FIO-8291

**Comment by lane-formio** [comment]
After review we don't expect to resource this to any of our developers in the near future. We always welcome any contributions to achieve the expected behavior and will review any PRs made to resolve.
